internal/rbd/rbd_util.go

```diff
@@ -206,7 +206,18 @@ var (
 			needRbdNbd: false,
 		},
 		librbd.FeatureNameExclusiveLock: {
-			needRbdNbd: true,
+			needRbdNbd: false,
```

Comment:
as there is no mandatory requirement for the nbd mounted here, unit tests also need to be corrected for this one.

Reply:
What you mean? `needRbdNbd` is already `false`

Reply:
```=== RUN   TestGetCephClientLogFileName/test_for_empty_id
=== CONT  TestValidateImageFeatures
    rbd_util_test.go:176: 
        	Error Trace:	rbd_util_test.go:176
        	Error:      	Error message not equal:
        	            	expected: "feature exclusive-lock requires rbd-nbd for mounter"
        	            	actual  : "feature journaling requires rbd-nbd for mounter"
        	Test:       	TestValidateImageFeatures
    rbd_util_test.go:176: 
        	Error Trace:	rbd_util_test.go:176
        	Error:      	Error message not equal:
        	            	expected: "feature exclusive-lock requires rbd-nbd for mounter"
        	            	actual  : "feature journaling requires rbd-nbd for mounter"
        	Test:       	TestValidateImageFeatures
--- FAIL: TestValidateImageFeatures (0.00s)
```
please check the [go test](https://github.com/ceph/ceph-csi/runs/4427733123?check_suite_focus=true#step:3:1544) failure you will get what needs to be changed